src/main/java/toraylife/mappetextras/modules/client/providers/ResolutionProvider.java

```diff
@@ -0,0 +1,26 @@
+package toraylife.mappetextras.modules.client.providers;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.nbt.NBTTagCompound;
+import toraylife.mappetextras.modules.client.ClientData;
+
+public class ResolutionProvider implements IClientDataProvider{
+    @Override
+    public NBTTagCompound getData() {
+        int height = Minecraft.getMinecraft().displayHeight;
```

Comment:
Почему бы не сделать так:
```suggestion
        Minecraft mc = Minecraft.getMinecraft();
        int height = mc.displayHeight;
```